package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -88,9 +89,17 @@ def parse(self, **kwargs):
             names = np.zeros(natoms, dtype=object)
             types = np.zeros(natoms, dtype=np.int)
             bonds = []
+            #Checks whether file contains periodic boundary box info 
+            fline = inf.readline()
+            try:
+                float(fline.split()[1])
+            except:
```

Comment:
Catching all exception classes, including keyboard interruptions, might not be desired